web/consts/consts.go

```diff
@@ -0,0 +1,82 @@
+// Package consts contains constant strings used on HTTP handlers
+package consts
+
+import (
+	"strings"
+	"unicode"
+)
+
+const (
+	// Accept is the canonical header name used for negotiating a MIME Type
+	// for the content of the Request response
+	Accept = "Accept"
+
+	// AcceptEncoding is the canonical name given to the header used
+	// to indicate compression options
+	AcceptEncoding = "Accept-Encoding"
+
+	// Allowed is the canonical header used to indicate the supported
+	// Methods.
+	Allowed = "Allowed"
+
+	// ContentEncoding is the canonical Content-Encoding header
+	ContentEncoding = "Content-Encoding"
+	// ContentLength is the canonical Content-Length header
+	ContentLength = "Content-Length"
+	// ContentType is the canonical Content-Type header
+	ContentType = "Content-Type"
+	// ETag is the canonical ETag header
+	ETag = "Etag"
+
+	// Location is the canonical name given to the header used
+	// to indicate a redirection.
+	Location = "Location"
+)
+
+const (
```

Comment:
Just for completeness sake. In 2022 we have the following http methods (RFC 9110):
```
GET	Transfer a current representation of the target resource.
HEAD	Same as GET, but do not transfer the response content.
POST	Perform resource-specific processing on the request content.
PUT	Replace all current representations of the target resource with the request content.
DELETE	Remove all current representations of the target resource.
CONNECT	Establish a tunnel to the server identified by the target resource.
OPTIONS	Describe the communication options for the target resource.
TRACE	Perform a message loop-back test along the path to the target resource.
```

so you are missing `TRACE`

Reply:
no `PATCH`?

Reply:
I think `PATCH` is still on a different RFC